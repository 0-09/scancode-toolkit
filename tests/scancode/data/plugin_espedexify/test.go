// Copyright 2017 Google Inc.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.


// Package licenseclassifier provides methods to identify the open source
// license that most closely matches an unknown license.
package licenseclassifier

import (
        "archive/tar"
        "bytes"
        "compress/gzip"
        "fmt"
        "html"
        "io"
        "math"
        "regexp"
        "sort"
        "strings"
        "sync"
        "unicode"

        "github.com/google/licenseclassifier/stringclassifier"
        "github.com/google/licenseclassifier/stringclassifier/searchset"
)

