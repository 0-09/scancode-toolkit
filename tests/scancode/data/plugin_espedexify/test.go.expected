// Copyright 2017 Google Inc.
//
// SPDX-License-Identifier: Apache-2.0

// Package licenseclassifier provides methods to identify the open source
// license that most closely matches an unknown license.
package licenseclassifier

import (
        "archive/tar"
        "bytes"
        "compress/gzip"
        "fmt"
        "html"
        "io"
        "math"
        "regexp"
        "sort"
        "strings"
        "sync"
        "unicode"

        "github.com/google/licenseclassifier/stringclassifier"
        "github.com/google/licenseclassifier/stringclassifier/searchset"
)

